aiida/tools/importexport/dbexport/__init__.py

```diff
@@ -489,7 +769,7 @@ def export_tree(
         progress_bar = get_progress_bar(total=len(given_entities), disable=silent)
         pbar_base_str = 'Preparing entities'
 
-    entries_to_add = dict()
+    entries_to_add = {}
```

Comment:
```suggestion
    entities_to_add = {}
```

Reply:
fixed in b689f5e6986c15ab18ad225e5659f1ef3bf8f9c0